### PR TITLE
Bugfix/add schedule holiday

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
**Are you a current Fivetran customer?** 

Internal
**What change(s) does this PR introduce?** 

Make downstream updates following the addition of the `schedule_holiday` table. Business resolution time calculations have been updated to account for holidays overlapping with ticket timelines.  This includes the `first_resolution_business_minutes`, `full_resolution_business_minutes`, `first_reply_time_business_minutes` fields. 

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [x] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)
Updated logic to existing fields

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [#92](https://github.com/fivetran/dbt_zendesk/issues/92)
- [ ] No 

**How did you test the PR changes?** 

 

- [x] Buildkite  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Comment by fivetran-reneeli** [comment]
- [ ] reminder to update docs

File: models/intermediate/int_zendesk__schedule_spine.sql [hunk 1]
```diff
@@ -88,21 +93,57 @@ with timezone as (
...
+            'second') }} /60
+        ) as holiday_end_time_from_week,
+        split_timezones.offset_minutes,
 
         -- we'll use these to determine which schedule version to associate tickets with
         split_timezones.valid_from,
         split_timezones.valid_until
 
     from schedule
+    join schedule_holiday
```
Comment (fivetran-jamie) [inline models/intermediate/int_zendesk__schedule_spine.sql:118]: should be a left join right?

File: models/resolution_times/int_zendesk__ticket_resolution_times_calendar.sql [hunk 2]
```diff
@@ -19,26 +19,75 @@ with historical_solved_status as (
...
+), ticket_schedule as (
+
+  select *
+  from {{ ref('stg_zendesk__ticket_schedule') }}
+
+), schedule_holiday as (
+
+    select *
+    from {{ ref('stg_zendesk__schedule_holiday') }}   
+
```
Comment (fivetran-jamie) [inline models/resolution_times/int_zendesk__ticket_resolution_times_calendar.sql:33]: why are we bringing these tables in here now? if we need to, is there a simpler way to bring in the holiday information?

File: models/intermediate/int_zendesk__schedule_spine.sql [hunk 3]
```diff
@@ -88,21 +93,57 @@ with timezone as (
...
+            dbt_date.week_start('schedule_holiday.holiday_start_date_at'), 
+            'schedule_holiday.holiday_start_date_at',
+            'second') }} /60
+        ) as holiday_start_time_from_week, -- number of minutes since the start of the week, where Sunday is the first day of the week
+
+        ( {{ dbt.datediff(
+            dbt_date.week_start('schedule_holiday.holiday_end_date_at'), 
+            'schedule_holiday.holiday_end_date_at',
+            'second') }} /60
+        ) as holiday_end_time_from_week,
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply models/intermediate/int_zendesk__schedule_spine.sql:110]: or rather `dbt_date.week_start` should truncate the holiday start date?

**Comment by fivetran-joemarkiewicz** [comment]
Closing this PR as PR #98 includes the relevant changes.

**Pull request closed without merging**